client/my-sites/themes/theme-showcase.scss

```diff
@@ -2,11 +2,83 @@
 @import "@wordpress/base-styles/mixins";
 
 body.is-section-themes-i4 {
-	--color-surface-backdrop: #fdfdfd;
+	&.theme-default.color-scheme {
+		--color-surface-backdrop: #fdfdfd;
+	}
 
-	&.theme-default .focus-content .layout__content {
+	.layout:not(.has-no-sidebar) .layout__content {
 		padding-top: var(--masterbar-height);
 	}
+
+	.theme__filters {
+		@include break-small {
+			padding: 24px 0;
+		}
+	}
+
+	.themes__selection .themes-list {
+		margin-top: 32px;
+	}
+
+	.upsell-nudge {
+		background-color: #f0f7fc;
+		border: none;
+		box-shadow: none;
+		margin-top: 32px;
+		margin-bottom: 0;
+		padding: 24px 30px;
+		border-radius: 4px;
+
+		&.is-dismissible {
+			.dismissible-card__close-button {
+				height: 16px;
+				width: 16px;
+
+				svg {
+					height: 16px;
+					width: 16px;
+				}
+			}
+		}
+
+		.banner__content {
+			gap: 16px;
+
+			@include breakpoint-deprecated( ">660px" ) {
+				gap: 32px;
+			}
+		}
+
+		.banner__title,
+		.banner__description {
+			color: var(--color-neutral-80);
+			font-size: $font-body-small;
+			line-height: 20px;
+		}
+
+		.banner__title {
+			font-weight: 500;
+		}
+
+		.banner__icon {
+			background-color: var(--studio-blue-50);
+			display: none;
+		}
+
+		.banner__icon-circle {
+			background-color: var(--studio-blue-50);
+		}
+
+		.banner__action .button {
+			background-color: var(--studio-blue-50);
+			border: none;
+			border-radius: 4px;
```

Comment:
Another "issue", not sure if it's intended or not: this PR somehow modifies the styles of the Upgrade button in the sidebar:

| Before | After |
| -- | -- |
| <img width="270" alt="image" src="https://user-images.githubusercontent.com/1525580/204437621-6f686183-efff-4c37-87e4-2935ea90a9b1.png"> | <img width="270" alt="image" src="https://user-images.githubusercontent.com/1525580/204437564-064dd739-eece-42c5-8ad6-972bfae447e9.png"> |

Reply:
Definitely not intended 🤦🏻‍♂️ thanks for noticing this!